src/pc/lua/utils/smlua_anim_utils.c

```
@@ -165,8 +177,9 @@ void smlua_anim_util_register_animation(const char *name, s16 flags, s16 animYTr
     }
 
     // Create a new node
-    struct CustomAnimation *node = calloc(1, sizeof(struct CustomAnimation));
+    struct CustomAnimation* node = calloc(1, sizeof(struct CustomAnimation));
     node->name = strdup(name);
```

Cppcheck (2.20.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)